src/SNPN/Common/INetworkService.cs

```diff
@@ -16,7 +16,8 @@ public interface INetworkService
 
 		Task<bool> ReplyToNotification(string replyText, string parentId, string userName, string password);
 
-		Task<ResponseResult> SendNotification(QueuedNotificationItem notificationItem, string token);
+		Task<ResponseResult> SendNotificationWNS(QueuedNotificationItem notificationItem, string token);
```

Comment:
Renamed the base WNS SendNotification just to be clear what it is vs the FCM version.